_posts/2020-10-21-announcing-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Announcing Presto Conference Tokyo 2020"
+author: Yuya Ebihara, LINE
+excerpt_separator: <!--more-->
+---
+
+On November 20, [Presto Conference Tokyo 2020 (Online)](https://techplay.jp/event/795265) will be hosted.
+Check out the details and register to attend!
+
+Last year, [Presto Conference Tokyo 2019]({% post_url 2019-07-11-report-for-presto-conference-tokyo %}) 
+was held in Japan with Martin Traverso, Dain Sundstrom and David Philips, 
+the founders of the Presto Software Foundation. 
+
+This year's event shows 6 sessions from Treasure Data, Amazon Web Services 
+Japan, Repro and LINE. Also, you can see an opening session by Martin.
+This is a valuable opportunity for those who are using Presto, for 
+data engineering, and those who don't use Presto yet but are interested in it, 
```

Comment:
This is a valuable opportunity to hear from engineers who are actually using Presto. It has something for those who are using Presto for data engineering and those who don't use Presto yet but are interested in it.